src/disco/css/Addon.scss

```diff
@@ -6,9 +6,20 @@ $primary-text-color: #000;
 $secondary-text-color: #6a6a6a;
 
 .addon {
+  align-items: center;
   background: #fff;
+  display: flex;
+  flex-direction: row;
+  line-height: 1.5;
   margin-top: 20px;
 
+  &.theme {
```

Comment:
I've generally looked to use CSS as it was intended, and find that's a simple approach generally. Two classes on an element e.g. `.addon.theme` seems to say what it needs to say. This is an addon that's also a theme.

If a class is on the "component/widget" and the related classes for things inside it are nested that generally works well. I've not used any of the naming schemes as you describe - I've not yet seen seen a scenario where they are necessary especially if the markup is right. That said we should not let historical usage prevent us from considering fresh approaches to CSS (this is something we can definitely talk about in London) and if we have a consensus that the something like BEM makes sense we could adopt it. But I feel I'd want to see that it really adds value before we jump to using something like that - my gut reaction is that it's overkill.

As long as nesting doesn't go too deep then for this project I'm fine with what we're doing - we could probably lint for that. What can be a problem with the likes of SASS and like any abstraction is that you can do bad things un-intentionally. E.g. you can over-nest and yet the src looks fine but the output is heavily over-specified. Again I think this is something we could use a CSS linter for to guard against and watch out for.